src/index.js

```diff
@@ -1,5 +1,5 @@
 import h from "./h"
 import app from "./app"
-import router from "./router"
+import Router from "./router"
```

Comment:
It's not important though since hyperapp isn't even aware of that name. I just think it's better for examples since it follows the usual conventions.